src/main/kotlin/org/rust/lang/utils/RustEscapesUtils.kt

```diff
@@ -32,7 +32,7 @@ fun parseRustStringCharacters(chars: String, outChars: StringBuilder, sourceOffs
 
     val outOffset = outChars.length
     var index = 0
-    chars.tokenize(RustEscapesLexer(STRING_LITERAL, true, true)).forEach {
+    chars.tokenize(RustEscapesLexer(STRING_LITERAL, true, true, true)).forEach {
```

Comment:
Here, we use `extendedByte = true` for `STRING_LITERAL`, but in the `of` method for `STRING_LITERAL` we don't pass any argument for `extendedByte` and it is thus `false`. This seems strange